Development.md

```diff
@@ -0,0 +1,115 @@
+# Introduction
+
+Welcome to the FeatBit development documentation.
+
+This document will guide you through the stages of setting up your development environment, connecting to our code base
+and finally submitting code to the project!
+
+# Get Started
+
+FeatBit consists of multiple services, to understand the architecture, please read
+our [document](https://featbit.gitbook.io/docs/tech-stack/architecture).
+
+![Architecture](https://2887964115-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FWMA5plqGXLhCIDCINvoc%2Fuploads%2Fn8WleMePXy4BW6b0qQq2%2Fimage.png?alt=media&token=27fd5b0e-e704-4e01-b084-b8f5399f2565)
+
+Most of the contribution work will focus on UI and API, we just need to set up their dependencies via docker compose and
+launch these two services from the code.
+
+## Setup dependencies
+
+Open [docker-compose.yml](./docker-compose.yml) and **comment out UI and api-server service**, they are the
+first two services. Then do
+
+```bash
+docker compose up -d
```

Comment:
Shouldn't we use docker-compose-dev.yml?

Reply:
done in f1b0a0b